src/redux/addCash.ts

```diff
@@ -239,14 +408,20 @@ const addCashGetTransferHash = (
       if (currentAccountAddress !== accountAddress) return;
 
       const destAssetAddress = toLower(
-        AddCashCurrencies[network][destCurrency]
+        AddCashCurrencies[network]?.[destCurrency]
       );
 
       if (transferHash) {
         logger.log('[add cash] - Wyre transfer hash', transferHash);
-        let asset = ethereumUtils.getAccountAsset(destAssetAddress);
+        let asset:
+          | ParsedAddressAsset
+          | undefined
+          | Pick<
```

Comment:
Should this type be co-located with the definition of `AddCashCurrencyInfo` rather than being duplicated here?